js/repeater.js

```diff
@@ -638,7 +645,7 @@
 						callbacks[stage](null);
 					}
 				};
```

Comment:
Grunt is complaining about these unnecessary semicolons. There's another on line 650. Symptom of converting from the variable-declared functions I imagine.